fmx_annealer.py

```diff
@@ -116,10 +116,52 @@ class FmxAnnealer(Device):
     inStatus = Cpt(EpicsSignalRO, '2}AnnealerIn-Sts') # status: 0 (Not In), 1 (In)
     outStatus = Cpt(EpicsSignalRO, '2}AnnealerOut-Sts') # status: 0 (Not In), 1 (In)
 
+    def anneal(self, anneal_time):
+        def status_callback(value, old_value, **kwargs):
+            if old_value == 0 and value == 1:
+                return True
+            else:
+                return False
+
+        status = SubscriptionStatus(self.inStatus, status_callback, run=False)
+        self.air.put(1)
+        status.wait(5)
+        time.sleep(anneal_time)
+
+        status = SubscriptionStatus(self.outStatus, status_callback, run=False)
+        self.air.put(0)
+        status.wait(5)
+
+
 class AmxAnnealer(Device):
     air = Cpt(EpicsSignal, '1}AnnealerAir-Sel')
     inStatus = Cpt(EpicsSignalRO, '2}AnnealerIn-Sts') # status: 0 (Not In), 1 (In)
 
+    def anneal(self, anneal_time):
+        def in_callback(value, old_value, **kwargs):
+            if old_value == 0 and value == 1:
+                logger.info(f'anneal state while annealing: {annealer.inStatus.get()}')
```

Comment:
since we are already getting the value for `inStatus`, the message should just show `{value}` for the value

Reply:
Good point.

Reply:
Removed these completely, I don't think they add anything if it's just showing a number.

Reply:
by using logging, time would be shown, which gives an idea how long things really take (not just the requested anneal time). but fair enough to take it out completely